project.md

```diff
@@ -58,11 +179,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think you'd be able to engage with the language quite frequently because the
+whole point is to create some way of transcribing taiko from something that is
+only heard to something that can be written down. There are already conventions
+in place that will make this easier, as shown in the link further above, but
+adjustments must still be made to allow for a fluent transcription.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+Because taiko is percussive, it's a less complicated to deal with than other
+instruments. I think it is very doable because it's a simple concept that I can
+always add onto if I have more time such as including dynamics, standard sheet
+music notation like repeats, and if I were really ambitious, trying to figure
+out a way to include the dancing aspect of taiko (taiko is not a stationary art;
+it is largely performance through dancing and shouting).
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+It's good because I have a lot of personal stake in this, and I believe it's
+fairly achievable. I would also have an ok pool of people I could ask feedback
+from (the taiko club) who I would consider as better than domain amateurs.
```

Comment:
Would you be able/willing to do user testing with some people in the club? I bet that they would have good insights for how your DSL should be designed!